deploy/catalog/templates/catalog.yaml

```diff
@@ -15,6 +15,24 @@ spec:
         imagePullPolicy: Always
         ports:
         - containerPort: 10000
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 10000
+          failureThreshold: 3
+          initialDelaySeconds: 5
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 2
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 10000
+          failureThreshold: 1
```

Comment:
I can honestly go either way. I was really only curious, not seeking any alteration to the PR. either approach (keeping just liveness, or including readiness as well) LGTM.